mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/Config.kt

```diff
@@ -82,6 +83,10 @@ data class Norg2Config(
     val baseUrl: String,
 )
 
+data class BrregConfig(
+    val baseUrl: String,
+)
```

Comment:
Vi har nå et par slike dumme dataklasser med bare `val baseUrl...`. Skal vi slå de sammen til en `BaseUrlConfig` eller noe sånt? Eller er det kjekt å ha de separat for lesbarhetens skyld?

Reply:
Mange av disse har også blitt definert som en egen dataklasse under sin respektive service/client i stedet for å definere det i `Config.kt`, f.eks.  `BrregClientImpl.Config( ...)`. Jeg synes dette blir ganske god dokumentasjon på hvilken type config hver respektive klient/service trenger, men det byr potensielt på litt kjedelig, repetativ kode.Alternativt som du sier blir å legge til en `BaseUrlConfig/HttpClientConfig`

Reply:
Godt poeng, det kan jeg ordne